applications/GeoMechanicsApplication/tests/test_partial_saturation.py

```diff
@@ -0,0 +1,87 @@
+import os
+
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.GeoMechanicsApplication.geomechanics_analysis as analysis
+
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+
+import test_helper
+
+class KratosGeoMechanicsPartialSaturation(KratosUnittest.TestCase):
+    """
+    This class contains benchmark tests which are checked with the analytical solution
+    """
+
+    def setUp(self):
+        # Code here will be placed BEFORE every test in this TestCase.
+        pass
+
+    def tearDown(self):
+        # Code here will be placed AFTER every test in this TestCase.
+        pass
+
+    def __test_saturated_below_phreatic_level_pw(self, test_name):
+        n_stages = 2
+
+        # get the parameter file names for all stages
+        file_path = test_helper.get_file_path(os.path.join('.', 'test_partially_saturated', test_name))
+        parameter_file_names = [os.path.join(file_path, 'ProjectParameters_stage' + str(i + 1) + '.json') for i in
```

Comment:
Python f-strings are the modern way to dynamically build strings:
```suggestion
        parameter_file_names = [os.path.join(file_path, f'ProjectParameters_stage{i+1}.json') for i in
```

Reply:
done.